neuralprophet/forecaster.py

```diff
@@ -340,6 +341,7 @@ def __init__(
         # General
         self.name = "NeuralProphet"
         self.n_forecasts = n_forecasts
+        self.q_hats = []
```

Comment:
As this variable is specific to CP, I think it should ideally not be a general class attribute.
Maybe we can make it a part of the uncertainty config? It is only estimated once and then retrieved, so the initialized config should be an OK place for it IMO.
(there, the name would also be less ambiguous, but maybe a slightly more verbose name may help, there, too)

Reply:
This is resolved, although with the conformal-specific config.